.env.example

```diff
@@ -122,3 +122,5 @@ AZURE_OPEN_AI_ENABLE_TEST_MODE=true
 
 # A comma separated list of emails that will be seeded as internal users
 DEMO_INTERNAL_USER_EMAILS=
+
+DEBUGBAR_ENABLED=false
```

Comment:
Added to ensure that development environments are not slowed when debugbar is not being used